module.nix

```diff
@@ -71,6 +71,13 @@ in
       disko = builtins.trace "the .disko output is deprecated, please use .diskoScript instead" (cfg.devices._scripts pkgs).diskoScript;
       diskoNoDeps = builtins.trace "the .diskoNoDeps output is deprecated, please use .diskoScriptNoDeps instead" (cfg.devices._scripts pkgs).diskoScriptNoDeps;
 
+      diskoImages = diskoLib.makeDiskImage {
```

Comment:
Is it intentional that the binding uses plural form (image**s**) while the function uses singular form?

Reply:
no :D but which one is better?

Reply:
it returns a path to a directory with possibly multiple imnages

Reply:
Is it too late yet to rename the function to plural form as well then? I don't image their being too many users yet, but what do i know 😅 an alias could be a good compromise if so?

Reply:
It doesn't had documentation so far. So I think it is rather safe.